docs-search/client-halogen/src/Docs/Search/App/SearchResults.purs

```diff
@@ -594,24 +594,28 @@ renderType = case _ of
       , renderType t2
       ]
 
-  ParensInType ty ->
+  ParensInType _ ty ->
     HH.span_
       [ HH.text "("
       , renderType ty
       , HH.text ")"
       ]
 
+  -- FIXME(ast)
+  Skolem _ _ _ _ _ -> HH.text "Skolem"
+  TUnknown _ _ -> HH.text "TUnknown"
```

Comment:
I may be wrong; but I have the impression that `Skolem` and `TUnknown` only show up during type checkin. I don't believe they exist in well formed `docs.json` files. 
But I would appreciate guidance on how to handle these.